libs/packages/components/src/lib/search-result-list/search-result-list.component.ts

```diff
@@ -20,4 +28,34 @@ export class SdsSearchResultListComponent {
    * Child Template to be used to display the data for each item in the list of items
    */
   @ContentChild('resultContent') resultContentTemplate: TemplateRef<any>;
+
+  data: any = {
```

Comment:
data should have type... title: string, message: string

Reply:
Since different types are assigned to this object, a type declaration here would cause the code to break.